.github/workflows/test-deployment.yaml

```
@@ -0,0 +1,44 @@
+---
+name: test-deployment
+
+permissions:
+  contents: read
+
+on: pull_request
```

Nitpicking, feels weird to see on at a lower level
```suggestion

on: pull_request

permissions:
  contents: read

```